doc/neps/conf.py

```diff
@@ -218,3 +221,14 @@
      author, 'NumPyEnhancementProposals', 'One line description of project.',
      'Miscellaneous'),
 ]
+
+# -----------------------------------------------------------------------------
+# Intersphinx configuration
+# -----------------------------------------------------------------------------
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/dev', None),
+    'numpy': ('https://numpy.org/devdocs', None),
+    'scipy': ('https://docs.scipy.org/doc/scipy/reference', None),
+    'matplotlib': ('https://matplotlib.org', None)
+}
+
```

Comment:
Might make sense to merge the changes to this file immediately, so that all future NEPs can use it.

Reply:
Put into https://github.com/numpy/numpy/pull/15733